CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+* v2.20.0
+  - Adds new custom timings support which enables Custom Timings to work with SPA's and regardless of page loads
+  - Adds an option `automaticPerformanceCustomTimings` to for tracking performance.measure calls as custom timings 
```

Comment:
"...for tracking `performance.measure` calls..."

there's a rogue 'to' in there I think!